cmp/grid/columns/Column.js

```diff
@@ -344,8 +347,16 @@ export class Column {
     }
 
     /**
-     * Produce a Column definition appropriate for AG Grid.
+     * @param {Record} record
+     * @param {Object} [agParams] - raw params from agGrid event, if available.
```

Comment:
I'm calling this method from `GridModel.beginEditAsync()`, where I want to eval for a record but don't have the `agParams` as it's not being triggered from a grid event.

I wasn't sure as to value of that last arg - I think we have it for consistency with other callbacks in our API that often are attached to agParams, but it might be worse to *sometimes* pass it to the fn than to never pass it at all. The fn is getting a lot of other info from the rec, store, col, etc.

Reply:
agree we should take the lowest common denominator approach on what gets passed to editable -- record/column/gridModel should be all we need